Formula/govendor.rb

```diff
@@ -0,0 +1,39 @@
+class Govendor < Formula
+  desc "Go vendor tool that works with the standard vendor file."
+  homepage "https://github.com/kardianos/govendor"
+  url "https://github.com/kardianos/govendor/archive/v1.0.3.tar.gz"
+  sha256 "7d7a032355ee4117822150db72a9db745d9a147a2e32972e58a65fe7545444f2"
+  head "https://github.com/kardianos/govendor.git"
+
+  depends_on "go"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
+
+    (buildpath/"src/github.com/kardianos/").mkpath
+    ln_sf buildpath, buildpath/"src/github.com/kardianos/govendor"
+    system "go", "build", "-o", bin/"govendor"
+  end
+
+  test do
+    # Can't run directly in testpath because `govendor` expects $GOPATH to be
+    # a "real" path & panics that /tmp is actually symlink to /private/tmp.
```

Comment:
It's not clear to me how the below changed that at all, so I'm not sure that's what actually going on. In any case, the directory handling looks much more complicated than it needs to be.